watertap/data/techno_economic/municipal_drinking.yaml

```
@@ -1,14 +1,13 @@
 default:  # needed some parameters as placeholder, so including cost params here
-  cap_basis_par:
-    value: 0.0403
-    units: M$
-  cap_exp_par:
-    value: 0.8657
-    units: dimensionless
-  capital cost units:
-    value: M$
-  vol units:
-    value: Mgallons/day
-  cost_function_form:
-    function f(flow_vol_in): cap_basis_par*(flow_vol_in)^cap_exp_par
-    units: $
+  capital_cost:
+    basis: flow_vol
+    cost_factor: TPEC
+    reference_state:
+      value: 1
+      units: Mgallons/day
+    capital_a_parameter:
+      value: 0.0403
```

```suggestion
      value: 0.0403e6
```
since the prior unit was `M$`?